src/types/token.cpp

```diff
@@ -0,0 +1,332 @@
+/**
+ * @file src/types/token.cpp
+ * @brief Implementation of class Literal.
+ * @copyright (c) 2017 Avast Software, licensed under the MIT license
+ */
+
+#include <algorithm>
+#include <cassert>
+#include <sstream>
+#include <stack>
+
+#include "yaramod/types/token.h"
+#include "yaramod/types/tokenstream.h"
+#include "yaramod/utils/utils.h"
+
+namespace yaramod {
+
+#define TABULATOR_LENGTH 8
```

Comment:
Is this define used anywhere?

Reply:
It is used in the `token_stream.cpp` and `token.cpp` files.